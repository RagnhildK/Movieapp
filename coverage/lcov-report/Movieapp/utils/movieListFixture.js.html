
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for Movieapp/utils/movieListFixture.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../prettify.css" />
    <link rel="stylesheet" href="../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../index.html">All files</a> / <a href="index.html">Movieapp/utils</a> movieListFixture.js</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>2/2</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>0/0</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>0/0</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>2/2</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
        <template id="filterTemplate">
            <div class="quiet">
                Filter:
                <input oninput="onInput()" type="search" id="fileSearch">
            </div>
        </template>
    </div>
    <div class='status-line high'></div>
    <pre><table class="coverage">
<tr><td class="line-count quiet"><a name='L1'></a><a href='#L1'>1</a>
<a name='L2'></a><a href='#L2'>2</a>
<a name='L3'></a><a href='#L3'>3</a>
<a name='L4'></a><a href='#L4'>4</a>
<a name='L5'></a><a href='#L5'>5</a>
<a name='L6'></a><a href='#L6'>6</a>
<a name='L7'></a><a href='#L7'>7</a>
<a name='L8'></a><a href='#L8'>8</a>
<a name='L9'></a><a href='#L9'>9</a>
<a name='L10'></a><a href='#L10'>10</a>
<a name='L11'></a><a href='#L11'>11</a>
<a name='L12'></a><a href='#L12'>12</a>
<a name='L13'></a><a href='#L13'>13</a>
<a name='L14'></a><a href='#L14'>14</a>
<a name='L15'></a><a href='#L15'>15</a>
<a name='L16'></a><a href='#L16'>16</a>
<a name='L17'></a><a href='#L17'>17</a>
<a name='L18'></a><a href='#L18'>18</a>
<a name='L19'></a><a href='#L19'>19</a>
<a name='L20'></a><a href='#L20'>20</a>
<a name='L21'></a><a href='#L21'>21</a>
<a name='L22'></a><a href='#L22'>22</a>
<a name='L23'></a><a href='#L23'>23</a>
<a name='L24'></a><a href='#L24'>24</a>
<a name='L25'></a><a href='#L25'>25</a>
<a name='L26'></a><a href='#L26'>26</a>
<a name='L27'></a><a href='#L27'>27</a>
<a name='L28'></a><a href='#L28'>28</a>
<a name='L29'></a><a href='#L29'>29</a>
<a name='L30'></a><a href='#L30'>30</a>
<a name='L31'></a><a href='#L31'>31</a>
<a name='L32'></a><a href='#L32'>32</a>
<a name='L33'></a><a href='#L33'>33</a>
<a name='L34'></a><a href='#L34'>34</a>
<a name='L35'></a><a href='#L35'>35</a>
<a name='L36'></a><a href='#L36'>36</a>
<a name='L37'></a><a href='#L37'>37</a>
<a name='L38'></a><a href='#L38'>38</a>
<a name='L39'></a><a href='#L39'>39</a>
<a name='L40'></a><a href='#L40'>40</a>
<a name='L41'></a><a href='#L41'>41</a>
<a name='L42'></a><a href='#L42'>42</a>
<a name='L43'></a><a href='#L43'>43</a>
<a name='L44'></a><a href='#L44'>44</a>
<a name='L45'></a><a href='#L45'>45</a>
<a name='L46'></a><a href='#L46'>46</a>
<a name='L47'></a><a href='#L47'>47</a>
<a name='L48'></a><a href='#L48'>48</a></td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">4x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">4x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import { configureStore } from "@reduxjs/toolkit";
import movieRatingReducer from "../redux/movieSlicer"
&nbsp;
const movies = {
    425909: {
      title: "Ghostbusters: Afterlife",
      posterPath: "/sg4xJaufDiQl7caFEskBtQXfD4x.jpg",
      overview:
        "When a single mom and her two kids arrive in a small town, they begin to discover their connection to the original Ghostbusters and the secret legacy their grandfather left behind.",
    },
    438695: {
      title: "Sing 2",
      posterPath: "/aWeKITRFbbwY8txG5uCj4rMCfSP.jpg",
      overview:
        "Buster and his new cast now have their sights set on debuting a new show at the Crystal Tower Theater in glamorous Redshore City. But with no connections, he and his singers must sneak into the Crystal Entertainment offices, run by the ruthless wolf mogul Jimmy Crystal, where the gang pitches the ridiculous idea of casting the lion rock legend Clay Calloway in their show. Buster must embark on a quest to find the now-isolated Clay and persuade him to return to the stage.",
    },
    460458: {
      title: "Resident Evil: Welcome to Raccoon City",
      posterPath: "/7uRbWOXxpWDMtnsd2PF3clu65jc.jpg",
      overview:
        "Once the booming home of pharmaceutical giant Umbrella Corporation, Raccoon City is now a dying Midwestern town. The company’s exodus left the city a wasteland…with great evil brewing below the surface. When that evil is unleashed, the townspeople are forever…changed…and a small group of survivors must work together to uncover the truth behind Umbrella and make it through the night.",
    },
    512195: {
      title: "Red Notice",
      posterPath: "/lAXONuqg41NwUMuzMiFvicDET9Y.jpg",
      overview:
        "An Interpol-issued Red Notice is a global alert to hunt and capture the world's most wanted. But when a daring heist brings together the FBI's top profiler and two rival criminals, there's no telling what will happen.",
    },
  };
&nbsp;
  export const mockStore = configureStore({
    reducer: {
      movieRatings: movieRatingReducer,
    },
    preloadedState: {
      movieRatings: {
        sessionID: "",
        username: "",
        nmbMovies: 0,
        loading: false,
        movies: movies,
        ratings: {},
        totalResults: {},
        sortedIDs: [],
        participants: [],
      },
    },
  });</pre></td></tr></table></pre>

                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank" rel="noopener noreferrer">istanbul</a>
                at Sun Jan 30 2022 18:21:24 GMT+0100 (sentraleuropeisk normaltid)
            </div>
        <script src="../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../sorter.js"></script>
        <script src="../../block-navigation.js"></script>
    </body>
</html>
    