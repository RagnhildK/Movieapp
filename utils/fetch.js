const baseURL =
  "https://api.themoviedb.org/3/movie/popular?api_key=72c828341c35299683ab545ba90e7f50&language=en-US&page=1";


export const getMovie = async (handleResponse) => {
  try {
    const response = await fetch(baseURL);
    const json = await response.json();
    handleResponse(json);
  } catch (error) {
    console.log(error);
  }
};
