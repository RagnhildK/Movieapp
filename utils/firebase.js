// Import the functions you need from the SDKs you need
import { initializeApp } from "firebase/app";
// import { getAnalytics } from "firebase/analytics";
import {
  getFirestore,
  doc,
  setDoc,
  collection,
  addDoc,
  where,
  query,
  getDoc,
  getDocs,
} from "firebase/firestore";

// TODO: Add SDKs for Firebase products that you want to use
// https://firebase.google.com/docs/web/setup#available-libraries

// Your web app's Firebase configuration
// For Firebase JS SDK v7.20.0 and later, measurementId is optional
const firebaseConfig = {
  apiKey: "AIzaSyCANKrut9VEYA6j9Erq9KEtCgekuNM8Cbs",
  authDomain: "movieapp-903cc.firebaseapp.com",
  projectId: "movieapp-903cc",
  storageBucket: "movieapp-903cc.appspot.com",
  messagingSenderId: "658249651052",
  appId: "1:658249651052:web:b2e8e233e3e95753025949",
  measurementId: "G-C4KJBS2XJY",
};

// Initialize Firebase
const app = initializeApp(firebaseConfig);
// const analytics = getAnalytics(app);

const db = getFirestore();

const UserColl = collection(db, "Users");
// const SessionColl = collection(db, "Participants");

export async function addUser(name) {
  const data = {
    username: name,
  };
  await addDoc(UserColl, data);
}

export async function checkIfUserExists(user) {
  const q = query(UserColl, where("username", "==", user));
  const a = await getDocs(q);
  return a.size != 0;
}

export async function addSession(name, sessionID, nmbMovies) {
  const data = {
    [name]: {},
  };
  const movies = {
    amount: nmbMovies,
  };
  await setDoc(doc(db, "Participants", sessionID), data);
  await setDoc(doc(db, "Movies", sessionID), movies);
}

export async function addParticipant(username, sessionID) {
  const data = {
    [username]: {},
  };
  await setDoc(doc(db, "Participants", sessionID), data, { merge: true });
}

export async function getNmbOfMovies(sessionID, handleResponse) {
  const a = doc(db, "Movies", sessionID);

  let movies;
  await getDoc(a).then((doc) => {
    movies = doc.data();
  });
  await handleResponse(movies);
}

export async function updateRatings(username, ratings, sessionID) {
  const data = {
    [username]: ratings,
  };
  await setDoc(doc(db, "Participants", sessionID), data, { merge: true });
}

export async function getRatings(sessionID, handleResponse) {
  const a = doc(db, "Participants", sessionID);
  let ratings;
  await getDoc(a).then((doc) => {
    ratings = doc.data();
  });
  await handleResponse(ratings);
}

// export async function checkIfUserExists() {
//   const docRef = doc(db, "Users", "");
//   const a = await getDoc();
//   console.log(a);
//   a.forEach((doc) => {
//     // doc.data() is never undefined for query doc snapshots
//     console.log(doc.id, " => ", doc.data());
//   });
// }
