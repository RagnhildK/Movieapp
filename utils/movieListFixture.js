import { configureStore } from "@reduxjs/toolkit";
import movieRatingReducer from "../redux/movieSlicer"

const movies = {
    425909: {
      title: "Ghostbusters: Afterlife",
      posterPath: "/sg4xJaufDiQl7caFEskBtQXfD4x.jpg",
      overview:
        "When a single mom and her two kids arrive in a small town, they begin to discover their connection to the original Ghostbusters and the secret legacy their grandfather left behind.",
    },
    438695: {
      title: "Sing 2",
      posterPath: "/aWeKITRFbbwY8txG5uCj4rMCfSP.jpg",
      overview:
        "Buster and his new cast now have their sights set on debuting a new show at the Crystal Tower Theater in glamorous Redshore City. But with no connections, he and his singers must sneak into the Crystal Entertainment offices, run by the ruthless wolf mogul Jimmy Crystal, where the gang pitches the ridiculous idea of casting the lion rock legend Clay Calloway in their show. Buster must embark on a quest to find the now-isolated Clay and persuade him to return to the stage.",
    },
    460458: {
      title: "Resident Evil: Welcome to Raccoon City",
      posterPath: "/7uRbWOXxpWDMtnsd2PF3clu65jc.jpg",
      overview:
        "Once the booming home of pharmaceutical giant Umbrella Corporation, Raccoon City is now a dying Midwestern town. The company’s exodus left the city a wasteland…with great evil brewing below the surface. When that evil is unleashed, the townspeople are forever…changed…and a small group of survivors must work together to uncover the truth behind Umbrella and make it through the night.",
    },
    512195: {
      title: "Red Notice",
      posterPath: "/lAXONuqg41NwUMuzMiFvicDET9Y.jpg",
      overview:
        "An Interpol-issued Red Notice is a global alert to hunt and capture the world's most wanted. But when a daring heist brings together the FBI's top profiler and two rival criminals, there's no telling what will happen.",
    },
  };

  export const mockStore = configureStore({
    reducer: {
      movieRatings: movieRatingReducer,
    },
    preloadedState: {
      movieRatings: {
        sessionID: "",
        username: "",
        nmbMovies: 0,
        loading: false,
        movies: movies,
        ratings: {},
        totalResults: {},
        sortedIDs: [],
        participants: [],
      },
    },
  });