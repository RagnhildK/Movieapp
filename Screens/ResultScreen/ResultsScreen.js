import React from "react";
import ResultMovie from "../../components/ResultMovie/ResultMovie";
import { Provider } from "react-native-paper";
import {
  setLoading,
  setTotalResults,
  addParticipant,
  resetResultScreen,
  sortTotalResults,
} from "../../redux/movieSlicer";
import {
  View,
  StyleSheet,
  Pressable,
  Text,
  ActivityIndicator,
} from "react-native";
import { useSelector, useDispatch } from "react-redux";
import { getRatings } from "../../utils/firebase";
import { useEffect } from "react";
import * as Colors from "../../styles/colors";

export default function ResultScreen({ navigation }) {
  const { sessionID, participants, sortedIDs, loading } = useSelector(
    (state) => state.movieRatings
  );

  const dispatch = useDispatch();

  const handlePress = () => {
    dispatch(setLoading(true));
    getRatings(sessionID, handleResponse);
  };

  const handleResponse = (response) => {
    dispatch(resetResultScreen());
    for (let user in response) {
      let userRatings = response[user];
      dispatch(addParticipant(user));
      for (let movieId in userRatings) {
        let rating = userRatings[movieId];
        dispatch(setTotalResults({ movieId: movieId, rating: rating }));
      }
    }
    dispatch(sortTotalResults());
    dispatch(setLoading(false));
  };

  useEffect(() => {
    getRatings(sessionID, handleResponse);
  }, []);

  return (
    <Provider>
      <View style={styles.container}>
        <Text>{participants} participants in this session</Text>
        <Pressable style={styles.button} onPress={() => handlePress()}>
          <Text style={styles.buttonText}> Refresh results</Text>
        </Pressable>
        {loading ? (
          <ActivityIndicator
            style={styles.loading}
            size="large"
            color="#0000ff"
          />
        ) : (
          <View className="MoviesListed">
            <View>
              {sortedIDs.map(([id, _]) => (
                <ResultMovie key={id} id={id} />
              ))}
            </View>
          </View>
        )}
        <Pressable
          style={styles.button}
          onPress={() => navigation.navigate("CreateOrJoinScreen")}
        >
          <Text style={styles.buttonText}> Go to start screen</Text>
        </Pressable>
      </View>
    </Provider>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center",
    backgroundColor: Colors.BEIGE,
    marginLeft: 20,
    marginRight: 20,
  },
  button: {
    backgroundColor: Colors.BROWN_RED,
    margin: 10,
    padding: 10,
    maxWidth: 300,
    borderRadius: 15,
  },
  buttonText: {
    color: Colors.WHITE,
    textAlign: "center",
  },
});
