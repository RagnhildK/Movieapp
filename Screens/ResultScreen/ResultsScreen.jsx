import React, { useEffect } from "react";
import ResultMovie from "../../components/ResultMovie/ResultMovie";
import { Provider, Headline, Chip, IconButton } from "react-native-paper";
import {
  setLoading,
  setTotalResults,
  addParticipant,
  resetResultScreen,
  sortTotalResults,
} from "../../redux/movieSlicer";
import {
  View,
  ScrollView,
  StyleSheet,
  Pressable,
  Text,
  ActivityIndicator,
  SafeAreaView,
} from "react-native";
import { useSelector, useDispatch } from "react-redux";
import { getRatings } from "../../utils/firebase";

import * as Colors from "../../styles/colors";

export default function ResultScreen({ navigation }) {
  const { sessionID, participants, sortedIDs, loading } = useSelector(
    (state) => state.movieRatings
  );

  const dispatch = useDispatch();

  const handlePress = () => {
    dispatch(setLoading(true));
    getRatings(sessionID, handleResponse);
  };

  const handleResponse = (response) => {
    dispatch(resetResultScreen());
    for (let user in response) {
      let userRatings = response[user];
      dispatch(addParticipant(user));
      for (let movieId in userRatings) {
        let rating = userRatings[movieId];
        dispatch(setTotalResults({ movieId: movieId, rating: rating }));
      }
    }
    dispatch(sortTotalResults());
    dispatch(setLoading(false));
  };

  useEffect(() => {
    getRatings(sessionID, handleResponse);
  }, []);

  return (
    <Provider>
      {loading ? (
        <ActivityIndicator
          style={styles.container}
          size="large"
          color="#0000ff"
        />
      ) : (
        <SafeAreaView style={styles.container}>
          <ScrollView>
            <View style={styles.rowContainer}>
              <Headline style={styles.heading1}>
                Participants in {sessionID}:
              </Headline>
              <IconButton
                style={styles.refreshButton}
                icon="refresh"
                color={Colors.WHITE}
                onPress={() => handlePress()}
              />
            </View>
            <Text>
              {participants.map((i) => (
                <Chip key={i} disabled={true} style={styles.chip}>
                  <Text style={styles.genresText}>{i}</Text>
                </Chip>
              ))}
            </Text>
            <View>
              {sortedIDs.map(([id, _]) => (
                <ResultMovie key={id} id={id} />
              ))}
            </View>
          </ScrollView>
          <Pressable
            style={styles.button}
            onPress={() => navigation.navigate("CreateOrJoinScreen")}
          >
            <Text style={styles.buttonText}> Go to start screen</Text>
          </Pressable>
        </SafeAreaView>
      )}
    </Provider>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center",
    backgroundColor: Colors.DARK_PURPLE,
    padding: 20,
  },
  rowContainer: {
    flexDirection: "row",
  },
  heading1: {
    color: Colors.ORANGE_LIGHT,
    fontSize: 24,
    // margin: 30,
    // textAlign: 'center',
    fontStyle: "italic",
  },
  chip: {
    marginHorizontal: 4,
    maxHeight: 48,
    lineHeight: 10, //usikker på om denne trengs
    backgroundColor: Colors.PURPLE_LIGHT,
  },
  genresText: {
    // usikker på om dette trengs - må se mer på chips og hvorfor de tar så mye plass
    padding: 0,
    margin: 0,
    color: Colors.BLACK,
  },
  refreshButton: {
    backgroundColor: Colors.PURPLE,
    alignSelf: "center",
  },
  button: {
    alignSelf: "flex-end",
    backgroundColor: Colors.PURPLE,
    margin: 10,
    paddingHorizontal: 30,
    padding: 15,
    maxWidth: 200,
    borderRadius: 10,
  },
  buttonText: {
    color: Colors.WHITE,
    textAlign: "center",
  },
});
