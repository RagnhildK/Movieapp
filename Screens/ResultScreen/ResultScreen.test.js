import React from "react";
import renderer from "react-test-renderer";
import ResultScreen from "./ResultsScreen";
import { Provider } from "react-redux";
import { mockStore } from "../../utils/movieListFixture";

jest.useFakeTimers()

it("ResultScreen snapshot", () => {
  const tree = renderer
    .create(
      <Provider store={mockStore}>
        <ResultScreen/>
      </Provider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
