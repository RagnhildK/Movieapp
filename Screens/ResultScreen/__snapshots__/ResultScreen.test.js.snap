// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultScreen snapshot 1`] = `
<View
  collapsable={false}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "backgroundColor": "#0f0b1c",
        "flex": 1,
        "justifyContent": "center",
        "padding": 20,
      }
    }
  >
    <RCTScrollView>
      <View>
        <View
          style={
            Object {
              "justifyContent": "flex-start",
              "marginHorizontal": 10,
            }
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
                "marginBottom": 10,
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#000000",
                    "fontFamily": "System",
                    "fontWeight": "400",
                  },
                  Object {
                    "textAlign": "left",
                  },
                  Array [
                    Object {
                      "textAlign": "left",
                    },
                    Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "System",
                      "fontWeight": "400",
                      "writingDirection": "ltr",
                    },
                    Array [
                      Object {
                        "fontSize": 24,
                        "letterSpacing": 0,
                        "lineHeight": 32,
                        "marginVertical": 2,
                      },
                      Object {
                        "alignSelf": "center",
                        "color": "#FF9C55",
                        "fontSize": 26,
                        "fontStyle": "italic",
                      },
                    ],
                  ],
                ]
              }
            >
              Participants in 
              
            </Text>
            <View
              accessibilityRole="button"
              accessibilityState={
                Object {
                  "disabled": undefined,
                }
              }
              accessible={true}
              focusable={true}
              hitSlop={
                Object {
                  "bottom": 6,
                  "left": 6,
                  "right": 6,
                  "top": 6,
                }
              }
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "overflow": "hidden",
                  },
                  Array [
                    Object {
                      "alignItems": "center",
                      "justifyContent": "center",
                      "margin": 6,
                      "overflow": "hidden",
                    },
                    Object {
                      "borderRadius": 18,
                      "height": 36,
                      "width": 36,
                    },
                    undefined,
                    Object {
                      "alignSelf": "flex-end",
                      "backgroundColor": "#352761",
                    },
                  ],
                ]
              }
            >
              <View>
                <Text />
              </View>
            </View>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "flexWrap": "wrap",
              }
            }
          />
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                },
                Object {
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "textAlign": "left",
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 0.87)",
                    "fontFamily": "System",
                    "fontWeight": "400",
                    "writingDirection": "ltr",
                  },
                  Array [
                    Object {
                      "fontSize": 24,
                      "letterSpacing": 0,
                      "lineHeight": 32,
                      "marginVertical": 2,
                    },
                    Object {
                      "color": "#FF9C55",
                      "fontSize": 26,
                      "fontStyle": "italic",
                      "marginTop": 10,
                    },
                  ],
                ],
              ]
            }
          >
            Results from the voting
          </Text>
        </View>
        <View />
      </View>
    </RCTScrollView>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "flex-end",
          "backgroundColor": "#352761",
          "borderRadius": 10,
          "margin": 10,
          "maxWidth": 200,
          "padding": 15,
          "paddingHorizontal": 30,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "textAlign": "center",
          }
        }
      >
         Go to start screen
      </Text>
    </View>
  </RCTSafeAreaView>
</View>
`;
